src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmFieldService.java

```diff
@@ -234,6 +235,101 @@ private void validateFieldName(String field, String headerField, int line, Set<S
         existing.add(field);
     }
 
+    /**
+     * Retrieve field information for the project from exported_objects.csv
+     * @param
+     * @return
+     */
+    public CdmFieldInfo retrieveFieldsFromCsv(Path exportedObjectsPath) throws IOException {
+        CdmFieldInfo fieldInfo = new CdmFieldInfo();
+
+        try (
+            Reader reader = Files.newBufferedReader(exportedObjectsPath);
+            CSVParser parser = new CSVParser(reader, CSVFormat.DEFAULT
+                    .withHeader(ExportObjectsInfo.CSV_HEADERS)
+                    .withTrim());
+        ) {
+            List<String> headers = parser.getHeaderNames();
+            for (String header : headers) {
```

Comment:
Since the CDM_ID field is required all over the place, I think we should translate the incoming record_id field into the dmrecord field. Or maybe we should have had the export command put the field in as dmrecord to begin with, which we could change in this PR. I think we are going to want to rename the field from dmrecord into record_id everywhere in the future, but we don't need to do that in this PR.